src/components/PlayerSearch.vue

```diff
@@ -10,63 +10,54 @@
       <h3 style="text-align:center">There were no results for the query : {{name}}</h3>
     </div>
     <div v-else>
-      <h1 v-show="Array.isArray(players.result)">
-        Search results for '{{name}}' (Page {{page}}/{{Math.ceil(players.totalCount/20)}}) :
-      </h1>
-      <div style="display:inline-block;margin-bottom:50px">
-        <v-btn
-          style="display:inline-block;
-          width:40%;
-          position:absolute;
-          left:0"
-          class="btn-text"
-          @click.native="changePage(-1)"
-          :disabled="page<=1 || wait">
-          <p block style="margin:auto">Previous</p>
-        </v-btn>
-        <v-btn inline-block
-          style="display:inline-block;
-          width:40%;
-          right:0;
-          height:20;
-          position:absolute;"
-          class="btn-text"
-          @click.native="changePage(1)"
-          :disabled="nextDisabled">
-          <p block style="margin:auto">Next</p>
-        </v-btn>
+    <v-container
+      class="d-flex flex-row m-6 justify-center flex-wrap"
+      outlined
+      tile>
+      <div v-for="(player, index) in players.result" :key="index" class="mr-6">
+          <PlayerTag :UUID="player.uuid"
+          :title="`${getRanks(player.uuid)}\n${player.lastSeenName}`"
+          class="playerTag"
+          @click.native="goTo(`/player/${player.uuid}`)">
+          </PlayerTag>
       </div>
-      <div block v-for="(player, index) in players.result" :key="index" class="row">
-        <v-btn block
-          left
-          style="display:inline;
-          width:100%;
-          height:20;"
-          class="btn-text"
-          @click.native="$router.push(`/player/${player.uuid}`)">
-          <v-icon>perm_identity</v-icon>
-          <p block style="margin:auto">{{getRanks(player.uuid)}} {{player.lastSeenName}}</p>
-        </v-btn>
-      </div>
-    </div>
+    </v-container>
+    <v-footer class="footer" v-show="Array.isArray(players.result)">
+      <v-btn @click="changePage(-1)" :disabled="page<=1">Back</v-btn>
+      <v-divider/>
+      <h3 style="font-size: max(1.4vw,9pt);">
+        Search results for '{{name}}' (Page {{page}}/{{Math.ceil(players.totalCount/50)}})
+      </h3>
+      <v-divider/>
+      <v-btn @click="changePage(1)" :disabled="page>=Math.ceil(players.totalCount/50)">Next</v-btn>
+    </v-footer>
+  </div>
   </div>
 </template>
 
 <style scoped>
   a {
     text-decoration: none;
   }
-  .btn-text {
-    float: left;
+  .playerTag:hover{
+    cursor: pointer;
+  }
+  .footer{
+    position:fixed;
```

Comment:
```suggestion
    position: fixed;
```
(same for the other properties)